_includes/share-article-list.html

```diff
@@ -0,0 +1,25 @@
+{% assign base_url = 'https://womencodingcommunity.com' %}
+{% assign post_url = page.url %}
+
+{% assign share_list = site.data.blog-share %}
+
+<ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdown-share-article">
+    {% for entry in share_list %}
+        <li class="d-flex align-tems-center">
```

Comment:
is this class correct name? `align-*tems*-center`

Reply:
thanks for the catch!